$MirOS: ports/www/dillo/v2/patches/patch-src_css_cc,v 1.1 2009/08/23 15:27:58 bsiegert Exp $
--- src/css.cc.orig	Mon Jan 18 18:00:19 2010
+++ src/css.cc	Sun Feb 21 18:17:01 2010
@@ -598,7 +598,7 @@ void CssContext::buildUserAgentStyle () 
 
 void CssContext::buildUserStyle () {
    Dstr *style;
-   char *filename = dStrconcat(dGethomedir(), "/.dillo/style.css", NULL);
+   char *filename = dStrconcat(dGethomedir(), "/.etc/dillo/style.css", NULL);
 
    if ((style = a_Misc_file2dstr(filename))) {
       CssParser::parse (NULL,NULL,this,style->str, style->len,CSS_ORIGIN_USER);
